examples/README.md

```diff
@@ -73,41 +86,42 @@ with the lane owner.
 |       | car#1 |       |X|       | Car#2 |      |
 |       |       |       |X|       |       |      |
 |       |       |       |X|       |       |      |
+```
 
 Here are the possible obstacles and the action you should take when hitting
 them:
 
-* obstacles.NONE        there is no obstacle at this position. You can return any
+- obstacles.NONE        there is no obstacle at this position. You can return any
                         action.
                         Score : 0
 
-* obstacles.PENGUIN     return actions.PICKUP and move 1 step forward. You will
+- obstacles.PENGUIN     return actions.PICKUP and move 1 step forward. You will
                         not move backward if you do not pick up the penguin.
                         Score : 10 for PICKUP otherwise 0
 
-* obstacles.WATER       return actions.BRAKE if you want to continue forward.
+- obstacles.WATER       return actions.BRAKE if you want to continue forward.
                         return actions.RIGHT or actions.LEFT if you like to
                         bypass the obstacle. Any other action will cause you to
                         move backward.
                         Score: 4 for BRAKE, 0 for LEFT/RIGHT otherwise -10
 
-* obstacles.CRACK       return actions.JUMP if you want to continue forward
+- obstacles.CRACK       return actions.JUMP if you want to continue forward
                         return actions.RIGHT or actions.LEFT if you like to
                         bypass the obstacle. Any other action will cause you to
                         move backward.
                         Score: 5 for JUMP, 0 for LEFT/RIGHT otherwise -10
 
-* obstacles.TRASH       you must return actions.RIGHT or actions.LEFT to bypass
+- obstacles.TRASH       you must return actions.RIGHT or actions.LEFT to bypass
                         the obstacle. Any other action will cause you to
                         move backward.
                         Score: 0 for LEFT/RIGHT otherwise -10
 
-* obstacles.BIKE        you must return actions.RIGHT or actions.LEFT to bypass
+- obstacles.BIKE        you must return actions.RIGHT or actions.LEFT to bypass
                         the obstacle. Any other action will cause you to
                         move backward.
                         Score: 0 for LEFT/RIGHT otherwise -10
 
-* obstacles.BARRIER     you must return actions.RIGHT or actions.LEFT to bypass
+- obstacles.BARRIER     you must return actions.RIGHT or actions.LEFT to bypass
                         the obstacle. Any other action will cause you to
                         move backward.
                         Score: 0 for LEFT/RIGHT otherwise -10
```

Comment:
I think this will look better using a table:
https://help.github.com/articles/organizing-information-with-tables/

Reply:
@nirs the kids will not use a MD viewer to read this. They will use Gedit, cat or Intellij.
Making this a table in MD format will be harder to read. They are using this information a lot.

Reply:
Good point - but since the kids must get the source from github, why not use the documentation on github?

We can easily publish the docs by creating a "docs" directory with some markdown files, making the docs available on RedHat-Israel.github.io/ROSE/.